tests/e2e/lib/blocks/mailchimp.js

```diff
@@ -31,7 +31,7 @@ export default class MailchimpBlock {
 	 *
 	 */
 	async connect( isLoggedIn = true ) {
-		const setupFormSelector = this.getSelector( "a[href*='marketing/connections']" );
+		const setupFormSelector = this.getSelector( "a[href*='wpcom-marketing-connections']" );
```

Comment:
Do we need to update this as well, even though the link remains the same?

Reply:
The final destination is the same, but the link in the `a` tag is not. I changed this to fix an e2e test that relies on this selector and was not finding the element when the link changed